project-koku/koku
# [COST-4028] recreate OCPPodSummaryByNodeP as partitioned table

## Jira Ticket

[COST-4028](https://issues.redhat.com/browse/COST-4028)

## Description

This change will create the `OCPPodSummaryByNodeP` table, correctly using `set_pg_extended_mode` and `unset_pg_extended_mode` in the migration.

This will also make the `0301` and `0310` migrations no-ops.

## Testing

1. run the migration
2. check to make sure the table is partitioned in the db:
```
postgres=# set search_path to org1234567 ;
SET
postgres=# \dt reporting_ocp_pod_summary_by_node_p
                                List of relations
   Schema   |                Name                 |       Type        |  Owner   
------------+-------------------------------------+-------------------+----------
 org1234567 | reporting_ocp_pod_summary_by_node_p | partitioned table | postgres
```

## Notes

...
